docs/reference/snapshot-restore/snapshot-vers-compat.asciidoc

```diff
@@ -0,0 +1,30 @@
+
+[cols="^,^,^,^,^"]
+|====
+| 4+^h| Cluster version
+h| Snapshot version         | 5.0–5.6    | 6.0–6.8    | 7.0–{prev-major-last} | 8.0–{minor-version}
```

Comment:
Could we clarify that by "snapshot version" we really mean the oldest index-created version? For instance a 7.16 cluster can hold indices created in 6.8 but if you take a snapshot of such a cluster you won't be able to fully restore it into an 8.x cluster.

Reply:
Thanks for pointing this out. After giving it some thought, I think it may be simpler to keep the concept of "snapshot version" and "index creation version" compatibility separate—even if the version compatibility looks the same.

Taking your example, a 7.16 snapshot can contain indices created in 7.16, 7.14, and 6.8. Based on my testing and understanding, you can safely restore the 7.16 and 7.14 indices from that snapshot to an 8.0 cluster but not the 6.8. indices. The snapshot is compatible; the 6.8 index is not.

I did make some additional changes to the index compatibility section, including adding a new table to focus on minor versions.

Let me know if anything is off the mark!